include/rest/api_client.hpp

```diff
@@ -144,19 +243,11 @@ namespace disccord
                     }
 
                     template <typename TModel>
-                    pplx::task<void> request_multi_json(route_info& route, std::vector<TModel> body, const pplx::cancellation_token& token = pplx::cancellation_token::none())
+                    pplx::task<void> request_multi_json(route_info& route, TModel body, const pplx::cancellation_token& token = pplx::cancellation_token::none())
```

Comment:
The general rundown is this:
- `request` is for requests to routes which do not expect a body or response
- `request_json` is for requests to routes which expect a response, a body or both
- `request_multi_json` is for requests to routes which expect an array of response models, body models or both
- `request_multipart` is for requests to routes which expect a multipart body, and optionally a response.